test/apps/iodemo/ucx_wrapper.h

```diff
@@ -68,12 +68,26 @@ class UcxLog {
 };
 
 
+/*
+ * Holds details of arrived AM message
+ */
+struct UcxAmDesc {
```

Comment:
can we avoid this struct and pass only relevant fields?

Reply:
you mean pass data and params as separate arguments? I'd better pass single parameter (as it is now), because IODEMO does nothing with it except passing to UcxWrapper recv function.

Reply:
if it does nothing, maybe it should opaque type which is only in ucx_wrapper.cc but not in ucx_wrapper.h?